### PR TITLE
documentation improvements for manual installation

### PR DESCRIPTION
**Pull request by Spitfireap**

### CODE REVIEW

File: doc/manual_installation/dovecot.rst [hunk 1]
```diff
@@ -10,19 +10,32 @@ suitable for this combination.
...
   # maildir
-  mail_location = maildir:~/.maildir
+  mail_location = maildir:~/Maildir
 
-Then, edit the ``inbox`` namespace and add the following lines::
+Then, edit the ``inbox`` namespace and add the following line::
 
   inbox = yes
 
+In order for dovecot to automaticaly create ususal folder on account creation, edit the ``inbox`` namespace of :file:`15-mailboxes.conf`::
```
Comment (tonioo) [inline doc/manual_installation/dovecot.rst:37]: usual folders

File: doc/manual_installation/dovecot.rst [hunk 2]
```diff
@@ -438,15 +468,6 @@ uncommented::
...
 
-Messages filtering using Sieve is done by the LDA.
-
-Inside :file:`conf.d/15-lda.conf`, activate the ``sieve`` plugin like this::
-
-  protocol lda {
-    # Space separated list of plugins to load (default is global mail_plugins).
-    mail_plugins = $mail_plugins sieve
-  }
-
```
Comment (tonioo) [inline doc/manual_installation/dovecot.rst:449]: Why do you remove that part?

File: doc/manual_installation/modoboa.rst [hunk 3]
```diff
@@ -169,7 +175,8 @@ you just have to run the following command:
    If you choose to install extensions one at a time, you will have to
    add their names in settings.py to ``MODOBOA_APPS``. Also ensure that
    you have the line ``from modoboa_amavis.settings import *`` at the
-   end of this file.
+   end of this file and possibly perform more steps. Check the manuall
```
Comment (tonioo) [inline doc/manual_installation/modoboa.rst:178]: manual

File: doc/manual_installation/webserver.rst [hunk 4]
```diff
@@ -239,25 +199,21 @@ Make sure to replace the ``<...>`` in the sample configuration with
...
-  at ``https://foo/modoboa`` then ``<modoboa's root url>`` is
+  at ``https://foo/modoboa`` then ``<hostname>`` is
   ``/modoboa``.  In this case you probably also have to set the
   ``alias`` directives to point to where Modoboa's sitestatic and
   media directories are because otherwise nginx won't be able to find
   them.
 
-  If Modoboa is at the root of your domain, then ``<modoboa root
-  url>`` is an empty string and can be deleted from the configuration
+  If Modoboa is at the root of your domain, then ``<hostnamel>``
```
Comment (tonioo) [inline doc/manual_installation/webserver.rst:211]: <hostname>

**Comment by Spitfireap** [comment]
@tonioo any clue why [these lines ](https://github.com/modoboa/modoboa/blob/b736c51d623f3c0f13c84bf43106d699ba407fa8/doc/upgrade.rst?plain=1#L216) don't show in the doc build ?

**Pull request merged**